running_local_operator.md

```diff
@@ -21,7 +21,7 @@ local operator will resolve its CR.
 
 ```
 oc get csv -n openstack-operators <your operator CSV> -o json | \
-  jq -r 'del(.metadata.generation, .metadata.resourceVersion)'  > operator_csv.json
+  jq -r 'del(.metadata.generation, .metadata.resourceVersion, .metadata.uid)'  > operator_csv.json
```

Comment:
But if it's the same instance of a CSV that you're backing up and restoring, then the UID should be the same.  If there were to be a conflict later, it would be because you had installed a new instance of the CSV since you originally made the backup, in which case I would think the backup is no longer relevant/needed.  Regardless, it doesn't hurt to remove the `.metadata.uid`, so this is fine.

Reply:
It's all good!  I'll merge it now.